packages/@okta/vuepress-site/docs/reference/api/devices/index.md

```diff
@@ -1014,19 +1014,19 @@ The device model defines several read-only properties:
    }
 }
 ```
+
 ### Device Status
 
-> More details on [Device Lifecycle](https://help.okta.com/oie/en-us/Content/Topics/identity-engine/devices/manage-device-lifecycle.htm)
+> More details on [Device Lifecycle](https://help.okta.com/okta_help.htm?type=oie&id=ext-devices-lifecycle)
```

Comment:
This link is not going to Device Lifecycle doc.

Reply:
Hey Van, you're correct. The original link was a 404. That page was split up into a few others. I'm now forwarding the alias to this page: https://help.okta.com/oie/en-us/Content/Topics/identity-engine/devices/devices-lifecycle.htm 

The alias was just created, so, I'd hope it'd be active. But we need to wait for another doc product build for it to be working.